lib/ansible/modules/cloud/vmware/vmware_guest_disk_facts.py

```diff
@@ -38,6 +38,12 @@
      description:
      - UUID of the instance to gather facts if known, this is VMware's unique identifier.
      - This is required parameter, if parameter C(name) is not supplied.
+   uuid_type:
```

Comment:
You are not doing anything with the type?
 Needs at least `self.uuid_type = self.params['uuid_type']` in `__init__`